### ISSUE TITLE
Add a geoscribble-notes layer

### ISSUE THREAD

**Issue by Zverik** [openstreetmap/iD#10204]
### Description

There's a panel under the "Map Data" button, "Data Layers". It contains OpenStreetMap Notes, for example.

I recently have made a GeoScribble service that stores drawings and short notes made with the Every Door editor. [They look like this](https://geoscribble.osmz.ru/#18/59.43530/24.74529)

In JOSM I currently use these notes by adding a [WMS layer](https://josm.openstreetmap.de/wiki/Maps/Worldwide#GeoScribbleMapNotes). It works, but not the best way, since it creates a load on the server, and the text would be distorted when scaling.

The best way would be to add a new "GeoScribbles" data layer, which accesses [/scribbles](https://geoscribble.osmz.ru/docs#/default/scribbles_scribbles_get) endpoint that returns all geometries and notes as json. And then display them on the map.

The layer doesn't have to be editable, it's just a digital walking papers thing. In the future we could add deleting obsolete drawings, but it's not important. I just want to see scribbles in the iD editor :)

**Comment by tordans** [comment]
@bhousel given your knowledge of both iD and Rapid, is https://github.com/facebook/Rapid/issues/1372 something that can be ported to iD with reasonably effort or is this part of the code in Rapid that changed too much?

**Comment by bhousel** [comment]
> @bhousel given your knowledge of both iD and Rapid, is [facebook/Rapid#1372](https://github.com/facebook/Rapid/issues/1372) something that can be ported to iD with reasonably effort or is this part of the code in Rapid that changed too much?

The service code to fetch features from the server should be mostly reusable, and the rendering code to draw the shapes would be mostly different.  But there are plenty of examples in iD's code that do similar things, so it should be pretty simple to implement.

**Comment by simonpoole** [comment]
@Zverik you should simply provide this (the geojson data) via a WFS service and WMS as a temp. solution for editors that don't support it. Add the later to ELI.

**Comment by Zverik** [comment]
@simonpoole good idea! I totally forgot about ELI, made a pull request: https://github.com/osmlab/editor-layer-index/pull/2312

GeoScribble already provides a WMS endpoint. Alas iD does not support custom overlay layers (see #9280).

**Comment by tyrasd** [comment]
I think the current solution with the WMS overlay on ELI is fine for now.
